docs/index.rst

```diff
@@ -6,6 +6,15 @@
 FESOM2 documentation
 ====================
 
+The Finite Element Sea Ice-Ocean Model (FESOM2).
+
+Multi-resolution ocean general circulation model that solves
+the equations of motion describing the ocean and sea ice using
+finite-element and finite-volume methods on unstructured
```

Comment:
remove finite-element, that's for FESOM 1 :)

Reply:
Done (updated paragraphs to balance the text & linebreaks)